imagedephi/base_rules.yaml

```diff
@@ -82,3 +82,615 @@ rules:
     - tag_name: Predictor
       method: keep
       type: metadata
+    - tag_name: Software
+      method: delete
+      type: metadata
+    - tag_name: ReferenceBlackWhite
+      method: keep
+      type: metadata
+    - tag_name: GPSVersionID
+      method: delete
+      type: metadata
+    - tag_name: '11'
+      method: delete
+      type: metadata
+    - tag_name: GPSMapDatum
+      method: delete
+      type: metadata
+    - tag_name: GPSLatitudeRef
+      method: delete
+      type: metadata
+    - tag_name: OldSubfileType
+      method: delete
+      type: metadata
+    - tag_name: Threshholding
+      method: delete
+      type: metadata
+    - tag_name: FillOrder
+      method: delete
+      type: metadata
+    - tag_name: DocumentName
+      method: delete
+      type: metadata
+    - tag_name: Make
+      method: delete
+      type: metadata
+    - tag_name: Model
+      method: delete
+      type: metadata
+    - tag_name: MinSampleValue
+      method: delete
+      type: metadata
+    - tag_name: MaxSampleValue
+      method: delete
+      type: metadata
+    - tag_name: PageName
+      method: delete
+      type: metadata
+    - tag_name: Xposition
+      method: delete
+      type: metadata
+    - tag_name: Yposition
+      method: delete
+      type: metadata
+    - tag_name: T4Options
+      method: delete
+      type: metadata
+    - tag_name: T6Options
+      method: delete
+      type: metadata
+    - tag_name: PageNumber
+      method: delete
+      type: metadata
+    - tag_name: GPSDateStamp
+      method: delete
+      type: metadata
+    - tag_name: GPSLatitude
+      method: delete
+      type: metadata
+    - tag_name: '306'
```

Comment:
Is this distinct from DateTime?

Reply:
It doesn't appear to be. The tiff standard doesn't mention anything else it could be, and there's only one tag in `tifftools.constants` with that number. I wrote a python script to generate rules from `unhandled.txt` and it looks like there was an extra line in that file, resulting in a redundant rule.

```
306 - DateTime
306 - 306
```

Reply:
And I generated the unhandled.txt file with some horrible bashisms, so it is probably my fault it is in there...